7.4/fpm/ubuntu/build/install_xdebug.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -x
+
+apt-get update && apt-get install php-pear
+
+pecl install xdebug-beta && docker-php-ext-enable xdebug
+
+echo xdebug.remote_enable=1 >> $PHP_INI_DIR/conf.d/docker-php-ext-xdebug.ini; \
+echo xdebug.remote_autostart=0 >> $PHP_INI_DIR/conf.d/docker-php-ext-xdebug.ini; \
+echo xdebug.remote_port=9000 >> $PHP_INI_DIR/conf.d/docker-php-ext-xdebug.ini; \
+echo xdebug.remote_connect_back=0 >> $PHP_INI_DIR/conf.d/docker-php-ext-xdebug.ini; \
+echo xdebug.remote_host=host.docker.internal >> $PHP_INI_DIR/conf.d/docker-php-ext-xdebug.ini; \
+echo xdebug.idekey=PHPSTORM >> $PHP_INI_DIR/conf.d/docker-php-ext-xdebug.ini; \
+echo xdebug.remote_log=/var/log/xdebug.log >> $PHP_INI_DIR/conf.d/docker-php-ext-xdebug.ini;
```

Comment:
It would be nice if this script would clean APT cache. This way the size of the image will be smaller.

Don't forget to remove `php7.4-dev` once everything is set :)

Reply:
Done.